.github/workflows/manual_distribution.yml

```diff
@@ -37,11 +37,8 @@ jobs:
             "staging": {
               "firebase_app_id": "1:1039839682638:android:bbcfa8c9939ee993ea631f"
             },
-            "schnapps": {
-              "firebase_app_id": "1:1039839682638:android:12d2ad31cc39093cea631f"
-            },
             "devnet": {
-              "firebase_app_id": "1:1039839682638:android:fdff563e24b69c5fea631f"
+              "firebase_app_id": "1:1039839682638:android:12d2ad31cc39093cea631f"
```

Comment:
We changed `schnapps` flavor name to `devnet` for consistency, but on DashPay devnet builds will be published to a different Firebase app.